dali/daliadmin/daadmin.cpp

```diff
@@ -274,19 +274,29 @@ void set(const char *path,const char *val)
 //=============================================================================
 
 void get(const char *path)
+{
+    getValue(path, nullptr);
+}
+
+void getValue(const char *path, StringBuffer *value)
 {
     StringBuffer head;
     StringBuffer tmp;
     const char *tail=splitpath(path,head,tmp);
     Owned<IRemoteConnection> conn = querySDS().connect(head.str(),myProcessSession(),RTM_LOCK_READ, daliConnectTimeoutMs);
     if (!conn) {
+        if (value) //called by esp service
```

Comment:
rather than contain difference/logic of differences in this function.
I would be better to change get (don't mind if renamed to getValue), to always fill a StringBuffer and change daliadmin cli to do the log "Value of ..."